test/boulder-config.json

```diff
@@ -76,7 +76,12 @@
               {
                 "ID": "1.3.6.1.4.1.44947.1.1.1",
                 "type": "id-qt-cps",
-                "qualifier": "http://cps.root-x1.letsencrypt.org"
+                "qualifier": "http://cps.letsencrypt.org"
+              },
+              {
+                "ID": "1.3.6.1.4.1.44947.1.1.1",
+                "type": "id-qt-unotice",
+                "qualifier": "This Certificate may only be relied upon by Relying Parties and only in accordance with the Certificate Policy found at https://letsencrypt.org/documents/"
```

Comment:
This looks wrong, unless CFSSL is behaving differently than I expect. It looks like this structure will result in the following ASN.1:

```
PolicyInformation{
  policyIdentifier: "1.3.6.1.4.1.44947.1.1.1",
  policyQualifiers: [
    { policyQualifierId: "id-qt-cps", qualifier: value },
  ]
}
PolicyInformation{
  policyIdentifier: "1.3.6.1.4.1.44947.1.1.1",
  policyQualifiers: [
    { policyQualifierId: "id-qt-unotice", qualifier: value },
  ]
}
```

What we want instead is:

```
PolicyInformation{
  policyIdentifier: "1.3.6.1.4.1.44947.1.1.1",
  policyQualifiers: [
    { policyQualifierId: "id-qt-cps", qualifier: value },
    { policyQualifierId: "id-qt-unotice", qualifier: value },
  ]
}
```